crash-handler-process/platforms/upload-window-win.cpp

```diff
@@ -360,13 +382,25 @@ void UploadWindow::windowThread()
 	UpdateWindow(upload_window_hwnd);
 	showButtons(false, false, false, false);
 
+	HFONT hFont = CreateFont(0, 0, 0, 0, FW_DONTCARE, FALSE, FALSE, FALSE, DEFAULT_CHARSET, OUT_DEFAULT_PRECIS, CLIP_DEFAULT_PRECIS, CLEARTYPE_QUALITY, DEFAULT_PITCH | FF_SWISS, L"Arial");
+	SendMessage (upload_label_hwnd, WM_SETFONT, WPARAM (hFont), TRUE);
+	SendMessage (ok_button_hwnd, WM_SETFONT, WPARAM (hFont), TRUE);
+	SendMessage (yes_button_hwnd, WM_SETFONT, WPARAM (hFont), TRUE);
+	SendMessage (cancel_button_hwnd, WM_SETFONT, WPARAM (hFont), TRUE);

```

Comment:
can true be used as well?

Reply:
it is a Windows API call so I prefer to use TRUE.